### PR TITLE
PY: Wrapper for get-set password provisioning

### PR DESCRIPTION
**Pull request by theisjabra**
Wrapper for get-set password provisioning.

Branch uses libjabra debug-build and will be merged into py-release branch until c-sdk is production ready.

### CODE REVIEW

File: nodesdk/src/main/device.cc [hunk 1]
```diff
@@ -1051,6 +1051,52 @@ Napi::Value napi_GetXpressUrl(const Napi::CallbackInfo& info) {
...
+
+            Jabra_ReturnCode retCode = Jabra_GetPasswordProvisioning(deviceId, buffer.data(), size);
+            if (retCode != Return_Ok) {
+                util::JabraReturnCodeException::LogAndThrow(functionName, retCode);
+                return std::string(); // Dummy return - avoid compiler warnings.
+            }
+
+            std::string result(buffer.data());
+            return util::toUtf8(result, functionName);
+        }, [](const Napi::Env& env, const std::string& cppResult) {
```
Comment (dlaezza-jabra) [inline nodesdk/src/main/device.cc:1095]: You can probably skip the C++ lambda here and just use `Napi::String::New`, but it doesn't really matter

File: nodesdk/src/main/device.ts [hunk 2]
```diff
@@ -1324,7 +1324,33 @@ export class DeviceType implements DeviceInfo, DeviceTiming, MetaApi {
...
+  
+    /**
+     * Get the password for provisioning
+     * @returns {Promise<string, JabraError>} - Resolves to password string on success,
+     *   rejects with `JabraError` if an error occurs.
+     */
+    getPasswordProvisioningAsync() : Promise<string> {
+      _JabraNativeAddonLog(AddonLogSeverity.verbose, this.getPasswordProvisioningAsync.name, "called with", this.deviceID);
+      return util.promisify(sdkIntegration.GetPasswordProvisioning)(this.deviceID).then((result) => {
+        _JabraNativeAddonLog(AddonLogSeverity.verbose, this.getPasswordProvisioningAsync.name, "returned");
```
Comment (dlaezza-jabra) [inline nodesdk/src/main/device.ts:1353]: You might want to log the returned value in the `then` callback, but again, it doesn't really matter

**Review by dlaezza-jabra — APPROVED** [review]
A couple of little things that don't matter anymore for a dying project like this

File: nodesdk/src/main/device.ts (hunk 2 above)
Comment (theisjabra) [reply nodesdk/src/main/device.ts:1353]: Good point, will add

File: nodesdk/src/main/device.cc (hunk 1 above)
Comment (theisjabra) [reply nodesdk/src/main/device.cc:1095]: I get `cannot determine which instance of overloaded function "Napi::String::New" is intended`

**Pull request merged**